File: go/fixture/go-mini/internal/models/status.go
package models

const (
	StatusReady    = "READY"
	StatusDegraded = "DEGRADED"
	StatusDown     = "DOWN"
	StatusBooting  = "BOOTING"
)

// IsValidStatus checks whether the status is valid.
func IsValidStatus(s string) bool {
	return s == "READY" || s == "DEGRADED" || s == "DOWN" || s == "BOOTING"
}

File: py/fixture/py-mini/internal/models/status.py
"""Device status values as they travel on the wire."""

STATUSES = ("READY", "DEGRADED", "DOWN", "BOOTING")
STATUS_READY, STATUS_DEGRADED, STATUS_DOWN, STATUS_BOOTING = STATUSES


def is_valid_status(s: str) -> bool:
    """Check whether the status is valid."""
    return s in STATUSES
